src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Movies = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Movies> = ({ movies }) => (
+  <>
+    <ul className="movies">
+      {movies.map((movie: Movie) => (
```

Comment:
ты уже типизировал что это массив Movie
```suggestion
      {movies.map((movie) => (
```